storage/engine.go

```diff
@@ -245,6 +247,8 @@ func (e *engine) initSQLDatabase() error {
 			return err
 		}
 		dialect = goose.DialectPostgres
+	case azuread.DriverName:
```

Comment:
I _prefer_ `"azuresql"` for consistency and so I don't have to check the value of something that never changes anyway